src/Transactions/TransactionFields/LinesField.php

```diff
@@ -59,20 +97,30 @@ public function addLine(TransactionLine $line)
     {
         Assert::isInstanceOf($line, $this->getLineClassName());
 
```

Comment:
With your change you can now also easily ensure that there is at most 1 total line. Multiple total lines are not accepted by Twinfield.

Reply:
Will add an assertion in `LinesField::addLine()` so that only 1 TOTAL line can be set.

Reply:
✓